src/main/java/io/antmedia/logger/GoogleAnalyticsLoggerImp.java

```diff
@@ -36,7 +36,7 @@ public void log(IThrowableProxy throwableProxy) {
         googleAnalytic.exception().
                 exceptionDescription(throwableStr).
                 clientId(instanceId).
-                send();
+                sendAsync();
     }
 
     @VisibleForTesting
```

Comment:
This string(UA-93263926-3) is also used in Launcher.java
Please define a contanst at Launcher.java and use that constant in here. 

Using same hard coded strings in different places generally increases the maintainability cost.